packages/block-editor/src/components/use-block-display-information/index.js

```diff
@@ -63,6 +65,7 @@ export default function useBlockDisplayInformation( clientId ) {
 				icon: match.icon || blockType.icon,
 				description: match.description || blockType.description,
 				anchor: attributes?.anchor,
+				customName: attributes?.metadata?.name,
```

Comment:
why does is this prop not called `name`?

Reply:
✅